src/components/v5/shared/ColonyVersionWidget/ColonyVersionWidget.tsx

```diff
@@ -14,12 +17,24 @@ const displayName = 'v5.ColonyVersionWidget';
 
 const ColonyVersionWidget: FC<ColonyVersionWidgetProps> = ({
   status,
-  lastVersion,
+  latestVersion,
   currentVersion,
 }) => {
   const { formatMessage } = useIntl();
   const isMobile = useMobile();
 
+  const {
+    actionSidebarToggle: [, { toggleOn: toggleActionSidebarOn }],
+  } = useActionSidebarContext();
+
+  const handleUpgradeColony = useCallback(
```

Comment:
I don't think `useCallback` is really necessary here, as the toggle action does not benefit from being memoized

Reply:
I've removed this now 👍